CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
  
 #### Fixes
 
+* [#1440](https://github.com/ruby-grape/grape/pull/1441): Remove pre-2.2.2 rubies from .travis.yml - [@jlfaber](https://github.com/jlfaber).
```

Comment:
Also, I wonder whether this change is worthy of a bump to 0.17.0 since it is technically a breaking change for people on older rubies.